.markdownlint-cli2.yaml

```diff
@@ -9,6 +9,7 @@ config:
   ol-prefix: false
   heading-start-left: false
   no-duplicate-heading: false
+  no-inline-html: false
```

Comment:
Why do we need to have inline HTML?

Reply:
Because of the Google Colab badge which I have added to `notebooks/README.md` following the suggestion by @rawkintrevo [here](https://github.com/instructlab/instructlab/pull/1053#issuecomment-2093132317)

Reply:
Ah I see, thanks for the explanation!